django/contrib/postgres/fields/jsonb.py

```diff
@@ -118,6 +118,24 @@ class KeyTextTransform(KeyTransform):
     _output_field = TextField()
 
 
+class CoerceToText(Transform):
```

Comment:
I think that this should be added in the separate pull request.

Reply:
I've removed this functionality, it seems the tests pass with it gone, so either something else changed in the interim or I didn't realise I'd changed something elsewhere.